apps/gcd/views/details.py

```diff
@@ -75,6 +91,94 @@ def show_publisher(request, publisher, preview=False):
+        if 'month' in request.GET:
+            year = int(request.GET['year'])
+            month = int(request.GET['month'])
+            # do a redirect, otherwise pagination links point to current month
+            return HttpResponseRedirect(
+              urlresolvers.reverse(date_type,
+                                   kwargs={'publisher_id': publisher_id,
+                                           'year': year,
+                                           'month': month} ))
+        if year:
+            year = int(year)
+        if month:
+            month = int(month)
+    except ValueError:
+        year = None
+
+    if year is None:
+        year = date.today().year
+        month = date.today().month
+
+    table_width = COVER_TABLE_WIDTH
+
+    covers = Cover.objects.filter(issue__series__publisher=publisher,
+                                  deleted=False).select_related('issue')
+    if use_on_sale:
+        covers = \
+          covers.filter(issue__on_sale_date__gte='%d-%02d-00' % (year, month),
+                        issue__on_sale_date__lte='%d-%02d-32' % (year, month))\
+                .order_by('issue__on_sale_date', 'issue__series')
+    else:
+        covers = \
+          covers.filter(issue__key_date__gte='%d-%02d-00' % (year, month),
+                        issue__key_date__lte='%d-%02d-32' % (year, month))\
+                .order_by('issue__key_date', 'issue__series')
+
+    start_date = datetime(year, month, 1)
+    date_before = start_date + timedelta(-1)
+    date_after = start_date + timedelta(31)
+    choose_url = urlresolvers.reverse(date_type,
+                                      kwargs={'publisher_id': publisher_id,
+                                              'year': year,
+                                              'month': month})
+    choose_url_before = urlresolvers.reverse(date_type,
+                                             kwargs={
+                                               'publisher_id': publisher_id,
+                                               'year': date_before.year,
+                                               'month': date_before.month})
+    choose_url_after = urlresolvers.reverse(date_type,
+                                            kwargs={
+                                              'publisher_id': publisher_id,
+                                              'year': date_after.year,
+                                              'month': date_after.month})
+
+    vars = {
+      'publisher': publisher,
+      'date': start_date,
+      'choose_url': choose_url,
+      'choose_url_after': choose_url_after,
+      'choose_url_before': choose_url_before,
+      'use_on_sale': use_on_sale,
+      'table_width': table_width,
+      'NO_ADS': True
+    }
+
+    return paginate_response(request, covers,
+      'gcd/details/publisher_monthly_covers.html', vars,
+      per_page=50,
+      callback_key='tags', callback=lambda page: get_image_tags_per_page(page))
```

Comment:
Any reason not to just do

callback=get_image_tags_per_page

since get_image_tags_per_page is already a callable of one argument?